website_sale_product_minimal_price/controllers/main.py

```diff
@@ -23,55 +22,25 @@ def get_combination_info_pricelist_atributes(
         today = fields.Date.today()
         res = []
         pricelist = current_website.get_current_pricelist()
-        pricelist_variant_items = pricelist.item_ids.filtered(
+        pricelist_items = pricelist.item_ids.filtered(
             lambda i: i.product_id.id == product.id
+            or i.product_tmpl_id.id == product.product_tmpl_id.id
+            or i.categ_id.id == product.categ_id.id
```

Comment:
On pricelist items, an upper category in the hierarchy includes all children categories, so this is not correct at all.